From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: fgei <fgei@gmail.com>
Date: Fri, 26 Jan 2024 14:57:11 +0000
Subject: [PATCH] temporary: Always partition third party storage

This will be removed in future milestone, but currently, sites can opt
out to this feature via the depreciation trial, see
https://developers.google.com/privacy-sandbox/3pcd/storage-partitioning#implementation_status
https://developers.google.com/privacy-sandbox/blog/storage-partitioning-deprecation-trial#disablethirdpartystoragepartitioning
---
 third_party/blink/common/features.cc                            | 2 +-
 .../blink/renderer/platform/runtime_enabled_features.json5      | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/third_party/blink/common/features.cc b/third_party/blink/common/features.cc
index 8c736ecb32ae7..c3f11bcce9819 100644
--- a/third_party/blink/common/features.cc
+++ b/third_party/blink/common/features.cc
@@ -697,7 +697,7 @@ BASE_FEATURE(kDiscardInputEventsToRecentlyMovedFrames,
 
 BASE_FEATURE(kDisableThirdPartyStoragePartitioning3DeprecationTrial,
              "DisableThirdPartyStoragePartitioning3DeprecationTrial",
-             base::FEATURE_ENABLED_BY_DEFAULT);
+             base::FEATURE_DISABLED_BY_DEFAULT);
 
 // Drop input events before user sees first paint https://crbug.com/1255485
 BASE_FEATURE(kDropInputEventsBeforeFirstPaint,
diff --git a/third_party/blink/renderer/platform/runtime_enabled_features.json5 b/third_party/blink/renderer/platform/runtime_enabled_features.json5
index 8ca258f76239a..668d7df6e70b6 100644
--- a/third_party/blink/renderer/platform/runtime_enabled_features.json5
+++ b/third_party/blink/renderer/platform/runtime_enabled_features.json5
@@ -1506,7 +1506,7 @@
       origin_trial_type: "deprecation",
       origin_trial_allows_insecure: true,
       origin_trial_allows_third_party: true,
-      status: "experimental",
+      status: "test",
       base_feature: "none",
       browser_process_read_write_access: true,
     },
